saleor/dashboard/product/forms.py

```diff
@@ -222,6 +222,7 @@ def delete(self):
 
 
 class ProductImageForm(forms.ModelForm):
+    use_required_attribute = False
```

Comment:
There was a problem, with browser validation of required fields. 
While on `dashboard` -> `product` -> `images`
I couldn't edit image's description, as it required me to upload a new photo instead.